### ISSUE TITLE
Alternative Repo

### ISSUE THREAD

**Issue by Jonovono** [callstack/react-native-opentok#35]
We started a new TokBox wrapper because this one didn't support things we wanted like changing camera position etc. At the time we started it this repo was pretty old too but now looks like it may be back in business? :p Anyways, we took a lot of inspiration from this one. Please let me know if you have any comments/concerns etc. We will be using it in production soon so plan on supporting it for the foreseeable future.

https://github.com/500LABS/HotBox

**Comment by masudhossain** [comment]
Quick question: Is there a way to specify the room name/number ? 

Reason: I have a web version that I'm currently running and I would like the rooms to be in sync. So if a user goes to tokbox-example.com/room1 on the web, I would want the react native user to be able to go to the same room1.

**Comment by Jonovono** [comment]
So you connect to a specific `sessionId` which usually represents a room like so:

`session.createSession(apiKey, sessionId, token)`

Usually, each person in that room (`sessionId`) gets their own `token`. 

When another user connects to that same room with the above method, you will get a call back from:

`subscriberDidConnect` which will pass along the `streamId` of that user. You use this to pass into the SubscriberView.

Hope that makes sense! We tried to model it around how OpenTok already works!

**Comment by timhuff** [comment]
I just tried for 2 hours to get callstack's solution to compile/run. It's poorly documented and the code seems to be somewhat messy. I gave this a shot and got video streaming on the first try. Seems like efforts should be focused on `react-native-hot-box`. The API is also more thorough/flexible. The downside is lack of Android support but I'm working on an app that might need to be migrated to Android soon. I'll definitely be contributing if need be.
